addons/planner_project/static/src/js/planner_project.js

```diff
@@ -0,0 +1,125 @@
+(function(){
```

Comment:
use the new way to load js (from GED), with `odoo.define()`
rebase your branch.